python/spear/config/default_config.wheeled_vehicle.yaml

```diff
@@ -39,22 +32,28 @@ OPENBOT:
       YAW: 0.0                     # Sonar relative orientation in [°]
       ROLL: 0.0                    # Sonar relative orientation in [°]
 
-  OPENBOT_WHEEL:
+  VEHICLE_COMPONENT:
+    MASS: 1.12                   # Mass of the vehicle chassis in [kg]
+    DRAG_COEFFICIENT: 1.0        # DragCoefficient of the vehicle chassis
+    CHASSIS_WIDTH: 15.0          # Chassis width used for drag force computation in [cm]
+    CHASSIS_HEIGHT: 15.0         # Chassis height used for drag force computation in [cm]
+
+  VEHICLE_WHEEL:
     AFFECTED_BY_HANDBRAKE: True       # Whether handbrake should affect this wheel
-    DAMPING_RATE: 0.25                # Damping rate for this wheel (Kgm^2/s) 
     MAX_BRAKE_TORQUE: 1500.0          # Max brake torque for this wheel (Nm)
```

Comment:
Are `MAX_BRAKE_TORQUE` and `MAX_HAND_BRAKE_TORQUE` required? I think these parameters are only required when using the high-level interface to scale the torque curve. Since we're directly setting torques, I don't think we need these.